src/backend/looped/vehicle/disable_siren.cpp

```diff
@@ -0,0 +1,29 @@
+#include "backend/looped_command.hpp"
+#include "natives.hpp"
+
+namespace big
+{
+	class siren_mute : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_tick() override
+		{
+			if (ENTITY::DOES_ENTITY_EXIST(self::veh))
+			{
+				VEHICLE::SET_VEHICLE_HAS_MUTED_SIRENS(self::veh, true); // Enables muted siren state
+			}
+		}
+
+		virtual void on_disable() override
+		{
+			if (ENTITY::DOES_ENTITY_EXIST(self::veh))
+			{
+				VEHICLE::SET_VEHICLE_HAS_MUTED_SIRENS(self::veh, false); // Disables muted siren state
+			}
+		}
+	};
+
+	siren_mute g_siren_mute("mutesiren", "Mute Siren", "Disables the siren sound of Emergency vehicles.",
```

Comment:
There is a dot in the end of the description, other commands don't have it for some reason.

Sorry for not saying it all in one review i'm blind.

Reply:
Ah, i added to end the sentence, my bad.

Reply:
Done